easybuild/easyconfigs/u/UDA/UDA-2.9.0-intel-compilers-2023.2.1.eb

```
@@ -0,0 +1,64 @@
+easyblock = 'CMakeNinja'
+
+name = 'UDA'
+version = '2.9.0'
+
+homepage = 'https://git.iter.org/projects/IMAS/repos/uda'
```

```suggestion
homepage = 'https://ukaea.github.io/UDA/'
```

I didn't get to access that other website?